MANIFEST.in

```diff
@@ -1,3 +1,6 @@
-include curl_cffi/_wrapper.*
-include curl_cffi/ffi/*
-include curl_cffi/include/curl/*
+recursive-include tests/*
```

Comment:
tests not included in sdist before this PR.
Also, Makefile and preprocess (which required for building from source) not included in sdist before this PR.